std/json/a_json_slow_init.go

```diff
@@ -16,9 +16,11 @@ func InternsOrThunks() {
 
 	jsonNamespace.InternVar("json-seq", json_seq_,
 		MakeMeta(
-			NewListFrom(NewVectorFrom(MakeSymbol("rdr"))),
+			NewListFrom(NewVectorFrom(MakeSymbol("rdr")), NewVectorFrom(MakeSymbol("rdr"), MakeSymbol("opts"))),
 			`Returns the json records from rdr as a lazy sequence.
-  rdr must be a string or implement io.Reader.`, "1.0"))
+  rdr must be a string or implement io.Reader.
+  Optional opts map may have the following keys:
+  :keywords? - if true, JSON keys will be converted from strings to keywords.`, "0.18.0"))
```

Comment:
`1.0` here too.

Reply:
I tried to squash these two commits using git rebase because this is such a small fix I don't think it deserves to have it's own commit. Unfortunately, I failed. We may talk about this next time